noxfile.py

```diff
@@ -133,6 +133,7 @@ def unit(session):
     session.install("opentelemetry-api==1.1.0")
     session.install("opentelemetry-sdk==1.1.0")
     session.install("opentelemetry-instrumentation==0.20b0")
+    session.run("python", "create_test_database.py")
```

Comment:
I really don't like that we have to create a database to run the unit tests. Is it worth separating out the logic for dynamically setting the config into another script?

So we have a new script that defines setting the config
````
def set_config(instance_id):
  ...
  
if __name__ == "__main__":
   main(sys.argv[1:])
````
And then the `create_test_database.py` script can import and use the function:
```
from new_script import set_config
```

WDYT?

Reply:
Yeah I agree, I think that's the right way to clean it up. PTAL.